jbehave-support-core/docs/Web-service.md

```diff
@@ -111,6 +111,25 @@ Given [CreateAccountRequest] data for [MYAPP]:
 | contract.contractParameters.billingDay | 7    |                                            |
 ```
 
+### Using XPaths in validation/saving response
+In case more flexible way to save or validate response is needed, XPaths can be used <br>
```

Comment:
```suggestion
In case more flexible way to save or validate response is needed, XPaths can be used. To use it simply start the value in the `name` column with `/`.
```

Just to make it a bit more clear and align with JSONPath md 🙂